src/rdkafka.h

```diff
@@ -8569,6 +8621,17 @@ RD_EXPORT
 rd_kafka_consumer_group_state_t rd_kafka_ConsumerGroupListing_state(
     const rd_kafka_ConsumerGroupListing_t *grplist);
 
+/**
+ * @brief Gets group_type for the \p grplist group.
+ *
+ * @param grplist The group listing.
+ *
+ * @return A group type.
+ */
+RD_EXPORT
+rd_kafka_consumer_group_type_t rd_kafka_ConsumerGroupListing_type(
+    const rd_kafka_ConsumerGroupListing_t *grplist);
```

Comment:
This should be a single `group` instead of `grplist`.

Reply:
Actually list here comes from 'listing' -> 'list', consumer_group_listing is better !